sycl/include/sycl/ext/oneapi/bfloat16.hpp

```diff
@@ -52,6 +57,7 @@ class bfloat16 {
     return static_cast<uint16_t>((intStorage + roundingBias) >> 16);
 #endif
   }
+
   static float to_float(const storage_t &a) {
```

Comment:
Might be a good idea to keep to_float implementation here also the same as that in devicelib.